src/coreclr/jit/instr.cpp

```diff
@@ -1789,7 +1789,7 @@ instruction CodeGen::ins_MathOp(genTreeOps oper, var_types type)
 }
 
 // Conversions to or from floating point values
-instruction CodeGen::ins_FloatConv(var_types to, var_types from)
+instruction CodeGen::ins_FloatConv(var_types to, var_types from, emitAttr attr)
```

Comment:
Done. It's required now because we divided INS_cvtsi2ss to INS_cvtsi2ss32 and INS_cvtsi2ss64. Similar for INS_cvtsi2sd. This is required because these instruction have some specific characteristic depending on if input is m32 or m64(W bit set or not)